### DISCUSSION TITLE
Is spectral IOR supported?

### DISCUSSION THREAD

**Discussion by gioarma** [mitsuba-renderer/mitsuba3#1006] [Q&A] [answered]
From the documentation it seems that the index of refraction of all BSDFs can only be specified as a single number, and not as a spectrum. Am I correct? It would be nice to have spectral support for IOR for modelling dispersion effects.
Thank you!

**Comment by njroussel** [comment, marked as answer]
Hi  @gioarma 

That is correct. ([Here'](https://github.com/mitsuba-renderer/mitsuba3/discussions/946)s an existing thread about this request).
